packages/rest/package.json

```diff
@@ -30,6 +30,7 @@
     "@loopback/express": "^3.1.3",
     "@loopback/http-server": "^2.3.5",
     "@loopback/openapi-v3": "^5.1.4",
+    "@loopback/router": "^0.0.1",
```

Comment:
I am bit reluctant to have a production-ready package like `@loopback/rest` to depend on a pre-1.0 version. I am proposing to release the first version of `@loopback/router` as `1.0.0`. APIs like `ControllerInstance` and `createControllerFactoryForClass` are re-rexported by `@loopback/rest` so we must them treat them as no-longer-experimental.

Thoughts?